### ISSUE TITLE
Translate currently typed word (where the cursor is located) to specific language and display in some place in the window.

### ISSUE THREAD

**Issue by rangerdio** [entertrained/entertrained-issues#15]
**Is your feature request related to a problem? Please describe.**
No problem.

**Describe the solution you'd like**
Would be nice translator to be added, so during typing if you think the word you type currently (where the cursor is) you do not understand, you can see the instant translation.

**Additional context**
for example:
![_2024-11-27_101953](https://github.com/user-attachments/assets/17f936e1-0212-4c2e-b66f-4c57ead918e5)

**Comment by caderek** [comment]
It's a nice idea, it would complement the current  built-in dictionary nicely (`Alt + Enter`), but there are some costs associated with it, so it will probably have to wait for now, until I figure out some monetization options (don't worry, all current features will stay free).

**Comment by rangerdio** [comment]
Thank you for the feedback, 
Yes, this would indeed be a valuable enhancement, especially as it would act as a translation feature, providing immediate and understandable feedback for non-native language writers.

The current functionality of providing definitions in English is already very helpful and minimizes the need to switch to external translator apps. However, incorporating a translation feature would further streamline the workflow by reducing the occasional need to leave the app altogether. Doesn't matter if it will like I mentioned above or with another key shortcut.

Perhaps leveraging an open-source solution like "LibreTranslate" could be a cost-effective way to implement this functionality.

**Comment by caderek** [comment]
Yes, I thought about LibreTranslate, and  I think I will probably go with it eventually, thank you for the suggestion.
